tests/spyre_util.py

```diff
@@ -363,12 +363,16 @@ def create_seq_prompt(model: ModelInfo, token_length: int) -> str:
     return tokenizer.decode(tokens)
 
 
-def create_random_request(request_id: int,
-                          num_tokens: int,
-                          sampling_params: SamplingParams,
-                          from_model_vocab: bool = False,
-                          model: Optional[ModelInfo] = None,
-                          seed: int = None) -> Request:
+def create_random_request(
+    request_id: int,
+    num_tokens: int,
+    sampling_params: SamplingParams,
+    from_model_vocab: bool = False,
+    model: Optional[ModelInfo] = None,
+    seed: int = None,
+    prefix_len: int = None,
+    n_lookahead: int = None,
```

Comment:
A small comment / docstring would be useful to explain these two parameters, this functions starts to become complex